examples/setixset.dx

```diff
@@ -0,0 +1,243 @@
+import sort
+
+
+'### Monoidal enforcement of uniqueness in sorted lists
+
+def last {n a} (xs:n=>a) : Maybe a =
```

Comment:
Lurking. Curious, what is this {} notation?

Reply:
It's new syntax that explicitly states all the variables that are implicit arguments to the function.  They used to be added automatically if the variables were lowercase.

Funnily enough, it was apparently partly motivated by Adam watching you getting confused when writing code like:
```
batch = Fin 10
def fun (batch=>Float) : ....
```
where `batch` in the function definition would silently shadow `batch` at the top level, because it started with a lower-case letter.